packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinSetupStrategies.tsx

```diff
@@ -1,94 +1,148 @@
-                    >
-                        Speed
-                        {strategy === 'speed' && <Icon icon="CHECK_ACTIVE" />}
-                    </SelectButton>
-                </Section>
-                <Section>
-                    <CoinjoinSessionDetail account={account} {...selectedStrategy} />
-                </Section>
-            </Content>
-            <Button
-                onClick={() => actions.startCoinjoinSession(account, selectedStrategy)}
-                icon="ARROW_RIGHT_LONG"
-                alignIcon="right"
-            >
-                Start coinjoin
-            </Button>
-        </Wrapper>
+        <>
+            <StyledCard>
+                <Heading>
+                    <Translation id="TR_START_ANONYMISATION" />
+                </Heading>
+                <Row>
+                    <StyledCryptoAmountWithHeader
+                        header={
+                            <AmountHeading>
+                                <Translation id="TR_AMOUNT" />
+                            </AmountHeading>
+                        }
+                        value={account.balance}
+                        symbol={account.symbol}
+                    />
+                    <StyledCryptoAmountWithHeader
+                        header={
+                            <AmountHeading>
+                                <Translation id="TR_SERVICE_FEE" />
+                            </AmountHeading>
+                        }
+                        value={fee}
+                        symbol={account.symbol}
+                        note={<Translation id="TR_SERVICE_FEE_NOTE" />}
+                    />
+                </Row>
+                <Line />
+                {isCustom ? (
+                    <CoinjoinCustomStrategy reset={reset} />
+                ) : (
+                    <CoinjoinDefaultStrategy strategy={strategy} setStrategy={setStrategy} />
+                )}
+            </StyledCard>
+
+            <StyledCard>
+                <ConfirmationsHeading>
+                    <Translation id="TR_CONFIRMATIONS" />
+                </ConfirmationsHeading>
+                <StyledCheckbox isChecked={connectedConfirmed} onClick={toggleConnectConfirmation}>
+                    <Translation id="TR_DEVICE_CONNECTED_CONFIRMATION" />
+                </StyledCheckbox>
+                <StyledCheckbox isChecked={termsConfirmed} onClick={toggleTermsConfirmation}>
+                    <Translation
+                        id="TR_TERMS_AND_PRIVACY_CONFIRMATION"
+                        values={{
+                            privacy: chunks => <Link href="https://trezor.io">{chunks}</Link>, // TODO: replace URL
+                            terms: chunks => <Link href="https://trezor.io">{chunks}</Link>, // TODO: replace URL
+                        }}
+                    />
+                </StyledCheckbox>
+            </StyledCard>
+
+            <Tooltip content={tooltip} interactive={false}>
```

Comment:
Use the `disabled` prop instead of the `tooltip` variable, simpler that way

Reply:
3770dcd2737b99b21f4c2d835c606a32271e4214